### ISSUE TITLE
Add role="alert" to the alert component

### ISSUE THREAD

**Issue by Veyfeyken** [digipolisantwerp/acpaas-ui_react#196]
## Problem

The alert component is not announced by screenreaders. role="alert" is missing. This role will create an implicit live region.

## Suggestion

Follow the ARIA design pattern for alert.

## Articles and demo

[ARIA Design Pattern for Alert](https://www.w3.org/TR/wai-aria-practices-1.2/#alert)
